Core/src/org/sleuthkit/autopsy/report/ArtifactSelectionDialog.java

```diff
@@ -66,21 +66,25 @@ public ArtifactSelectionDialog(java.awt.Frame parent, boolean modal) {
     @SuppressWarnings("deprecation")
     private void populateList() {
         try {
-            ArrayList<BlackboardArtifact.ARTIFACT_TYPE> doNotReport = new ArrayList<>();
-            doNotReport.add(BlackboardArtifact.ARTIFACT_TYPE.TSK_GEN_INFO);
-            doNotReport.add(BlackboardArtifact.ARTIFACT_TYPE.TSK_TOOL_OUTPUT);  // output is too unstructured for table review. 
-
-            artifacts = Case.getCurrentCase().getSleuthkitCase().getBlackboardArtifactTypesInUse();
+            ArrayList<BlackboardArtifact.Type> doNotReport = new ArrayList<>();
+            doNotReport.add(new BlackboardArtifact.Type(BlackboardArtifact.ARTIFACT_TYPE.TSK_GEN_INFO.getTypeID(),
+                    BlackboardArtifact.ARTIFACT_TYPE.TSK_GEN_INFO.getLabel(),
+                    BlackboardArtifact.ARTIFACT_TYPE.TSK_GEN_INFO.getDisplayName()));
+            doNotReport.add(new BlackboardArtifact.Type(BlackboardArtifact.ARTIFACT_TYPE.TSK_TOOL_OUTPUT.getTypeID(),
+                    BlackboardArtifact.ARTIFACT_TYPE.TSK_TOOL_OUTPUT.getLabel(),
+                    BlackboardArtifact.ARTIFACT_TYPE.TSK_TOOL_OUTPUT.getDisplayName())); // output is too unstructured for table review
+
+            artifacts = Case.getCurrentCase().getSleuthkitCase().getArtifactTypesInUse();
             artifacts.removeAll(doNotReport);
-            Collections.sort(artifacts, new Comparator<BlackboardArtifact.ARTIFACT_TYPE>() {
+            Collections.sort(artifacts, new Comparator<BlackboardArtifact.Type>() {
```

Comment:
ah, ok, I assumed artifact types were too. Then I would use Comparator.comparing(BlackboardArtifact.Type::getDisplayName) here, but that is mostly stylistic.